app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -82,13 +83,45 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/size_3x"
         android:layout_marginEnd="@dimen/size_3x"
-        android:layout_marginBottom="@dimen/size_10x"
+        android:layout_marginBottom="@dimen/size_7_5x"
         android:enabled="true"
         android:gravity="center"
-        android:padding="14dp"
+        android:padding="@dimen/size_2x"
```

Comment:
In zepplin there is 14dp:
![image](https://user-images.githubusercontent.com/3758285/222990614-ccbf64e5-83c6-4fed-869b-1ff0989c9230.png)

On my phone text is cut off from the bottom:
![image](https://user-images.githubusercontent.com/3758285/222990659-7beb4f46-a1da-48b0-884e-98f3558718ec.png)